includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -860,14 +860,14 @@ public function setup_fields_sections() {
 				'input_type'    => 'checkbox',
 				'default_value' => $default_settings['enable_ocr'],
 				'description'   => __(
-					'Detect text in an image and store that as post content',
+					'OCR detects text in images (i.e., handwritten notes) or PDF files and saves that as post content.',
```

Comment:
Hi @jeffpaul,

This text update is correct as per the given Google docs. but, I think OCR setting here is only related to images as we already have separate settings for scanning/reading PDFs. Can we get it updated only for images in google docs or any suggestions? 

![image](https://user-images.githubusercontent.com/10613171/154706999-5838c338-1296-4ae8-a10a-c4aaf5a10470.png)

Thanks.

Reply:
@iamdharmesh do you have a screenshot showing the separate settings for scanning/reading PDFs as you noted?

Reply:
![image](https://user-images.githubusercontent.com/10613171/155060522-59a797c2-adb7-40de-9a6a-be987fb73d59.png)